### PR TITLE
Yacht Dice: Textual fixes and changes (Docs / yaml description)

### PR DESCRIPTION
**Pull request by spinerak**
## What is this fixing or adding?
I've changed several textual... texts.
Most importantly, I've changed the setup guide (there was a broken link in there too; ../player-options).
I've made it much simpler, not explaining the entire AP workings.

## How was this tested?
I've tried and tested it via webhost.

## The new setup guide will look like this:
image

### CODE REVIEW

**Comment by spinerak** [comment]
ah damn I messed up and added other stuff to this PR, hold....

**Comment by spinerak** [comment]
I reverted it, ok should be good again......

File: worlds/yachtdice/docs/setup_en.md [hunk 1]
```diff
@@ -3,19 +3,13 @@
 ## Required Software
 
 - A browser (you are probably using one right now!).
-- Archipelago from the [Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases).
 
 ## Playing the game
 Open the Yacht Dice website. There are two options:
-- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open player.html in your browser. 
+- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser. 
 - Cruise over to the [Yacht Dice website](https://yacht-dice-ap.netlify.app/). This also works on mobile. If the website is not available, use the first option.
```
Comment (palex00) [inline worlds/yachtdice/docs/setup_en.md:10]: I would reverse these options so the website (the easier way) is listed first.

Maybe explain why one would prefer the one over the other?

Comment (spinerak) [reply worlds/yachtdice/docs/setup_en.md]: Changing the order makes sense!
I'm not sure how to explain why it's easier, I think it follows from the fact it's only one step :)
Thanks for the feedback!

File: worlds/yachtdice/docs/setup_en.md [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
 ## Playing the game
 Open the Yacht Dice website. There are two options:
-- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser. 
-- Cruise over to the [Yacht Dice website](https://yacht-dice-ap.netlify.app/). This also works on mobile. If the website is not available, use the first option.
+- Cruise over to the [Yacht Dice Website](https://yacht-dice-ap.netlify.app/). This is the easiest option. If the website is unavailable, use the next option.
+- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser.
 
 Press Archipelago, and after logging in, you are good to go. The website has a built-in client, where you can chat and send commands.    
-Both options also have an "offline" play option to try out the game without having to generate a game first.
+The website also have an "offline" play option to try out the game without having to generate a game first.
```
Comment (palex00) [inline worlds/yachtdice/docs/setup_en.md:13]: Why this change? Is it because the download is "technically" already offline? I'd still say it's fair to say you can do it offline with the downloaded version tbh.


If you want to keep this change though you need to change have to has.

Comment (spinerak) [reply worlds/yachtdice/docs/setup_en.md]: Fair! I've changed it to:
Both options also have a "Solo play" mode
Which I think makes sense because that's what the mode is called on the website.

**Review by palex00 — APPROVED** [review]
All concerns were addressed. Looks good. Fixes erronous information in the Options.py, too.

**Comment by spinerak** [comment]
I've updated the description of the PR: I've added a screenshot of what the setup page looks to make review easier.
@NewSoupVi would you be able to review this for me please? It's the last open PR for now that would really help give Yacht Dice the best introduction possible :)

**Review by NewSoupVi — APPROVED** [review]
